packages/inxi/build.sh

```diff
@@ -1,17 +1,17 @@
-TERMUX_PKG_HOMEPAGE=https://github.com/smxi/inxi
+TERMUX_PKG_HOMEPAGE=https://smxi.org/site/about.htm#inxi
 TERMUX_PKG_DESCRIPTION="Full featured CLI system information tool"
 TERMUX_PKG_LICENSE="GPL-3.0"
 TERMUX_PKG_MAINTAINER="@termux"
-TERMUX_PKG_VERSION="3.3.31-2"
-TERMUX_PKG_SRCURL=https://github.com/smxi/inxi/archive/refs/tags/${TERMUX_PKG_VERSION}.tar.gz
-TERMUX_PKG_SHA256=ff5d138392ac557e31ede6cf96d73d1b9f972f42f6529d47fec2c51184bff338
+TERMUX_PKG_VERSION="3.3.36-1"
```

Comment:
Is it appropriate to have dash/hyphen in version number? I am not familiar with apt versions. ArchLinux replaces some characters with dots https://wiki.archlinux.org/title/PKGBUILD#pkgver

Reply:
We got plenty of packages that wrongfully use hypen as version numbers anyway

Reply:
Welp that is how upstream is doing it.
We could substitute it.
But there's really no reason to.